src/v/coproc/event_handler.h

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright 2021 Vectorized, Inc.
+ *
+ * Licensed as a Redpanda Enterprise file under the Redpanda Community
+ * License (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * https://github.com/vectorizedio/redpanda/blob/master/licenses/rcl.md
+ */
+
+#pragma once
+
+#include "coproc/logger.h"
+#include "coproc/script_dispatcher.h"
+#include "coproc/wasm_event.h"
+#include "seastarx.h"
+
+#include <seastar/core/coroutine.hh>
+#include <seastar/core/future.hh>
+
+#include <absl/container/btree_map.h>
+#include <absl/container/btree_set.h>
+
+namespace coproc::wasm {
+
+class event_handler_exception final : public std::exception {};
```

Comment:
Because we do not need msg for this exception. Because we use vlog + throw for interrupt process. I thought  use custom exception is better than runtime_error

Reply:
done